editoast/Cargo.toml

```
@@ -118,7 +118,7 @@ serde_with = { version = "3.15.0", default-features = false, features = [
 stdext = "0.3.3"
 strum = { version = "0.26.3", features = ["derive"] }
 syn = "2.0"
-tempfile = "3.23.0"
+tempfile = "^3.20.0" # redis-test (crate cache) isn't up to date
```

You can remove the caret since it's [the default strategy](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements).

```suggestion
tempfile = "3.20.0" # redis-test (crate cache) isn't up to date
```

---

Good point 🤦